src/mono/browser/debugger/BrowserDebugProxy/DevToolsHelper.cs

```diff
@@ -293,7 +293,10 @@ internal sealed class MonoCommands
         public bool silent { get; set; }
         public bool returnByValue { get; set; } = true;
 
-        public MonoCommands(string expression) => this.expression = expression;
+        public MonoCommands(string expression)
+        {
+            this.expression = $"{expression} //# sourceURL=cdp://debug/eval.cdp";
```

Comment:
What is picking up `sourceURL` from this comment in the expression?

Reply:
the browser. Then when we receive a `Debugger.scriptParsed` with `url=cdp://debug/eval.cdp` and not empty as it is when we don't pass `cdp://debug/eval.cdp`